<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>628</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>kCal</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="kexue_Cal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>科学计算器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="king_Cal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="text">
     <string>kCal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ce">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>ce</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输入:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输出:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_inc">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_add">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cleft">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cright">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_output">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_exp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ln">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_log">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_square">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_root2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x^1/2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cube">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x^3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_root3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x^1/3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_m_inc">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cot">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_power">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_pi">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>pi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tanh">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tanh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cosh">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cosh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sinh">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sinh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_equal">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_M">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>kexue_Cal</zorder>
   <zorder>king_Cal</zorder>
   <zorder>pushButton_ce</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_c</zorder>
   <zorder>pushButton_add</zorder>
   <zorder>pushButton_inc</zorder>
   <zorder>pushButton_mul</zorder>
   <zorder>pushButton_div</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_m_add</zorder>
   <zorder>pushButton_point</zorder>
   <zorder>pushButton_sign</zorder>
   <zorder>pushButton_cleft</zorder>
   <zorder>pushButton_cright</zorder>
   <zorder>lineEdit_output</zorder>
   <zorder>label_show_exp</zorder>
   <zorder>button_ln</zorder>
   <zorder>button_log</zorder>
   <zorder>button_cos</zorder>
   <zorder>button_sin</zorder>
   <zorder>button_square</zorder>
   <zorder>button_root2</zorder>
   <zorder>button_cube</zorder>
   <zorder>button_root3</zorder>
   <zorder>pushButton_m</zorder>
   <zorder>button_tan</zorder>
   <zorder>button_m_inc</zorder>
   <zorder>button_cot</zorder>
   <zorder>button_exp</zorder>
   <zorder>button_power</zorder>
   <zorder>button_pi</zorder>
   <zorder>button_tanh</zorder>
   <zorder>button_cosh</zorder>
   <zorder>button_sinh</zorder>
   <zorder>pushbutton_equal</zorder>
   <zorder>label_M</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="import_statics"/>
    <addaction name="export_statics"/>
    <addaction name="separator"/>
    <addaction name="file_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置(S)</string>
    </property>
    <widget class="QMenu" name="theme_change">
     <property name="title">
      <string>主题(T)</string>
     </property>
     <addaction name="theme_black_white"/>
     <addaction name="theme_blue"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>背景图片(B)</string>
     </property>
     <addaction name="background_heros"/>
     <addaction name="background_graph"/>
    </widget>
    <addaction name="theme_change"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="help_H"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="import_statics">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>导入(I)</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="export_statics">
   <property name="text">
    <string>导出(O)</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="theme_black_white">
   <property name="text">
    <string>黑白(W)</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="theme_blue">
   <property name="text">
    <string>蔚蓝(B)</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="background_heros">
   <property name="text">
    <string>英雄海报(H)</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="background_graph">
   <property name="text">
    <string>自定义...(I)</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="file_quit">
   <property name="text">
    <string>退出(Q)</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="help_H">
   <property name="text">
    <string>帮助(H)</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>file_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
